.rubocop.yml

```diff
@@ -10,6 +10,7 @@ Style/RescueModifier:
 Metrics/BlockLength:
   Exclude:
     - test/**/*_test.rb
```

Comment:
Do we still need to exclude this?

Reply:
nope!  good catch :)

PS there might be more minitest remnants, I think I got them all but not sure